classes/module/Module.php

```diff
@@ -3489,6 +3497,38 @@ public function saveDashConfig(array $config)
     {
         return false;
     }
+
```

Comment:
Could you add two unit tests for these new methods ?

Reply:
done 👍  I tested the getter and the setter in the same method (so that I can check that I get the value I set before)